lib/jsrt/jsrtinternal.h

```diff
@@ -131,7 +131,7 @@ JsErrorCode ContextAPIWrapper(Fn fn)
     Js::ScriptContext *scriptContext = currentContext->GetScriptContext();
     try
     {
-        AUTO_NESTED_HANDLED_EXCEPTION_TYPE((ExceptionType)ExceptionType_JavascriptException);
```

Comment:
Thought about that, but didn't want  to add without knowing if we will get OOS here. I will add if I see a reason to add it.